### ISSUE TITLE
[bug] Critical Conflict with Polymer based mods.

### ISSUE THREAD

**Issue by ZombieNW** [Siphalor/nbt-crafting#87]
**Version**
1.18.1 - [1.18-pre1] 2.1.0
**Describe the bug**
When trying to connect to the server with bot nbtcrafting and any mod using the Polymer library, misc errors are thrown and you are unable to join the server.

**To Reproduce**
1. Join server with both nbtcrafting and any polymer based fabric mod (able to provide one if needed)
2. Kicked from the server
![image](https://user-images.githubusercontent.com/59242209/146622615-8960a430-555d-4bad-90bc-14be68413bcf.png)

Here is an image of an error message provided, after restarting the server without making any changes it changed to nbtcrafting:brewing, and then after another restart it went back to anvil.

**Expected behavior**
Being able to join the server with any polymer-based mod.

**Comment by Siphalor** [comment]
Is Nbt Crafting installed on the server?

Joining non-NbtC servers is currently officially unsupported.

**Comment by ZombieNW** [comment]
Yes it is installed on the server it's installed only on the server and the polymer mod is also installed on the server.

**Comment by Siphalor** [comment]
Ok, I'm now back at Minecraft modding and looking through these issues again.

Can you tell me which mods were installed on the server and do you still have the full server-side log?

**Comment by ZombieNW** [comment]
Here is the latest.log
[latest.log](https://github.com/Siphalor/nbt-crafting/files/7786646/latest.log)

The server boots just fine, but the server immediately kicks you on joining for a variety of reasons, from brewing, to anvils, to no message at all.
